tests/util/zookeeper_test.py

```diff
@@ -338,3 +338,17 @@ def test_get_topics(self, mock_client):
                 },
             }
             assert actual_without_fetch_state == expected_without_fetch_state
+
+            zk.get_children = mock.Mock(side_effect=NoNodeError())
+            actual_with_no_node_error = zk.get_topics()
+            expected_with_no_node_error = {}
+            zk.get_children.assert_called_with("/brokers/topics")
+            assert actual_with_no_node_error == expected_with_no_node_error
+
+    def test_get_brokers(self, mock_client):
```

Comment:
I think it's good idea to cover happy-case as well for get_brokers if it's not covered.